src/WireReader.cs

```diff
@@ -293,7 +293,7 @@ public TimeSpan ReadTimeSpan32()
         public IPAddress ReadIPAddress(int length = 4)
         {
             var address = ReadBytes(length);
-            return new IPAddress(address);
+            return new IPAddress(address.ToArray());
```

Comment:
Why the change? `address` is already a byte array.

Reply:
Sorry will remove, played with Span<byte>, but this gives not much for using in this project.
see System.Memory

Reply:
Yes, `Span` could be helpful.  However, it is not natively available for all the targets we support.